### ISSUE TITLE
PlayerRecipeBookClickEvent does not exist on Spigot

### ISSUE THREAD

**Issue by SlimeDog** [pop4959/Bolt#55]
Spigot 1.19.4-3744
Bolt 1.0.480

On server start-up
```
[07:21:56] [Server thread/INFO]: [Bolt] Enabling Bolt v1.0.480
[07:21:56] [pool-15-thread-1/INFO]: Loaded 0 block protections in 1.322 ms
[07:21:56] [pool-15-thread-1/INFO]: Loaded 0 entity protections in 0.168 ms
[07:21:56] [Server thread/ERROR]: [Bolt] Plugin Bolt v1.0.480 has failed to register events for class org.popcraft.bolt.listeners.adapter.PlayerRecipeBookClickListener because com/destroystokyo/paper/event/player/PlayerRecipeBookClickEvent does not exist.
```

No issues on Paper 1.19.4-526

**Comment by pop4959** [comment]
I'll take a look at fixing this one tonight. I must have missed the error when testing Spigot, as it should not be trying to register this event.

I don't think this will have any negative impact on using the plugin however, it is just the message.

**Comment by pop4959** [comment]
Fixed by 27cb3a68c429642b5c4eaf33a0674c7bc276cdef
